src/main/java/mate/academy/spring/model/dto/response/UserResponseDto.java

```diff
@@ -0,0 +1,40 @@
+package mate.academy.spring.model.dto.response;
+
+public class UserResponseDto {
+    private Long id;
+    private String email;
+    private String password;
+    private byte[] salt;
```

Comment:
I think password and salt it's too sensitive data to include them to ResponseDto

Reply:
You mean that sensitive data like password should be send only in authentication service? And here, in UserService we don't need it 'cause here we operate only with user object and his orders, and we have no need to do smth with his password?